### ISSUE TITLE
FailureDiffs directory along side the ReferenceImages

### ISSUE THREAD

**Issue by hugobast** [ashfurrow/Nimble-Snapshots#68]
Currently the FailureDiffs isn't configured but it could easily be configured with the defaults suggested [here](https://github.com/facebook/ios-snapshot-test-case#installation-with-cocoapods).

If it's worth anything to you guys let me know and I'll try to make it happen. I use the env var and it works fine but I had to do a lot of digging to finally see what the difference is between this folder that we aren't getting by default and the other one which we are.

**Comment by ashfurrow** [comment]
Neat, didn't know about that! I don't see any reason we shouldn't include this feature, would love to review a PR together 👍

**Comment by marcelofabri** [comment]
just to be clear, you mean a directory in the same level as ReferenceImages, not inside it, right? Something like this:

```
- Tests
   - ReferenceImages
   - FailureDiffs
```

**Comment by diogot** [comment]
Would be awesome if it is in the same directory, but with `failed` appended to file name.

**Comment by marcelofabri** [comment]
@diogot I disagree as it'd be easy to mistakenly commit them

**Comment by diogot** [comment]
I didn't thought about this, I agree with you.

**Comment by orta** [comment]
Aye, I'm with both of you, don't think we should set this to be something that could end up in git

**Comment by hugobast** [comment]
@marcelofabri yes absolutely on the same level as `ReferenceImages`. I'll open a PR. Thanks guys!

**Comment by orta** [comment]
@hugobast to my knowledge, the last three comments were saying that this shouldn't be added ( at least mine was ) as it is more likely to be annoying to people than the other way around.

If it is, then it should be off by default at least - the diff files are temporary data, whereas the reference images should be in the repo

**Comment by marcelofabri** [comment]
My comment was that it's a nice feature, but the tree should be something like this:

```
- Tests
   - ReferenceImages
   - FailureDiffs
```

And NOT this:
```
- Tests
   - ReferenceImages
       - spec.png
       - spec_failed.png
```

**Comment by hugobast** [comment]
Right that's the version I plan on implementing:

```
- Tests
   - ReferenceImages
   - FailureDiffs
```

I'm gonna tackle it this coming Friday.

**Comment by hugobast** [comment]
Maybe a README entry will be enough to educate that `Tests/FailureDiffs` should be added to `.gitignore`?
